packages/ember-engines/addon/templates/link-to-external-template.hbs

```diff
@@ -0,0 +1,2 @@
+<LinkTo @route={{this.route}} @isExternal={{true}} ...attributes>{{yield}}</LinkTo> 
```

Comment:
@BobrImperator personally I think this solves most but not all the ways `<LinkTo />` (and subsequently `<LinkToExternal />`) components can be used. e.g. it does not support providing query params or model for the external route.

we believe `<LinkToExternal />` should have exact same interface as `<LinkTo />` and re-implementing that interface in `ember-engines` is brittle and would lead to a lot of duplication.

For that reason, RFC https://github.com/emberjs/rfcs/pull/819 was opened to solve this problem in a way that would not require duplication nor use of intimate API.

cc @villander @runspired

Reply:
I agree with @SergeAstapov given the `LinkToExternal` is part of the first Ember Engines RFC we need to approve the new one to accept this implementation. That being said the RFC is the best way to build things right not just quickly. 

I appreciate your contribution @BobrImperator and would love to get you involved in the RFC https://github.com/emberjs/rfcs/pull/819

Reply:
Thanks for the feedback.
I forgot about all other args... would need to proxy them.

If you don't see it as feasible then it's fine with me. My main concern however is getting rid of the legacy components hack that I shouldn't have added in the first place. But had no better idea at the time.

Although, fixing it this way should allow people to more easily migrate to ember 4, in fact the reason I took another look at this was that embroider has disabled it's tests against engines because there was no support at the time while technically there's support finally but it's really hard to use at the moment.

https://github.com/embroider-build/embroider/pull/1123

Reply:
@BobrImperator as https://github.com/emberjs/rfcs/pull/819 takes some time to mature and land, using `@ember/legacy-built-in-components` as temp solution is totally fine as this allows community to use `ember-engines` with Ember.js 4+ until long-term solution lands.

there is nothing necessarily "bad" about `@ember/legacy-built-in-components`, it's just an extra code that needs to be shipped, but will get rid of it once https://github.com/emberjs/rfcs/pull/819 lands and implemented.